src/platform/forms-system/src/js/review/SubmitButtons.jsx

```diff
@@ -19,150 +22,54 @@ export default function SubmitButtons(props) {
     submission,
     formConfig,
   } = props;
+
   const appType = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
-  let submitButton;
-  let submitMessage;
 
   if (submission.status === false) {
-    submitButton = (
-      <ProgressButton
-        onButtonClick={onSubmit}
-        buttonText={`Submit ${appType}`}
-        buttonClass="usa-button-primary"
-      />
-    );
+    return <Default appType={appType} onBack={onBack} onSubmit={onSubmit} formConfig={formConfig} />;
```

Comment:
Oh wow, so I missed passing the formConfig through. I figured it was something like that. Thank you for fixing!!

Reply:
Yup! And I think we can move towards passing in only the things which matter most vs. all the things. Future iterations. 😄